ROADMAP.md

```diff
@@ -119,6 +120,7 @@
 - Alon Lavie
 - Lucia Specia
 - [Jaime Carbonell](https://github.com/machinetranslate/machinetranslate.org/issues/106)
+- Ondřej Bojar
```

Comment:
Good one!  Any others in that vein?

Reply:
I could add Martin Popel. He's recently had an MT paper published in Nature Communications.

Reply:
Ok, generally the co-authors at https://aclanthology.org/people/m/martin-popel/ is a good list.

In terms of priorities, I am a bit biased towards 
-  time-tested, i.e. when we're looking back we can say this was a once-a-decade contribution (e.g. BLEU, SMT, Transformer, MarianMT...)
- builders (e.g. CTO of Head of Research of Unbabel or SYSTRAN, key early Google Translate engineers, Jörg Tiedemann)
- "meta", e.g. Barry Haddow

Because publishing inherently generates a lot of noise, and because Western Europe and certain sectors of North America have a low output of products relative to PhDs and papers, there's a lot that's "hiding in plain sight", e.g. hardly anybody can name one person at Tencent Translate or Yandex Translate - which serve ~1 billion users - or anybody who built and exited a machine translation company.